ckanext/password_policy/helpers.py

```diff
@@ -58,7 +58,7 @@ def requirements_message(password_length=None, username=None):
 
     return _('Your password must be {} characters or '
              'longer and contain uppercase, lowercase, '
-             'digit and special character. '
+             'digit, and special character ( [ !#$%&\'()*+,-./[]^_`{{|}}~@" ).'
```

Comment:
I think the formatting is throwing me off, or why is the `[`-character in there twice?

Reply:
implemented